index.js

```diff
@@ -142,7 +142,7 @@ export default function typescriptProvider({negotiateProtocol}) {
 						],
 						ignoredByWatcherPatterns: [
 							...ignoredByWatcherPatterns,
-							...Object.values(relativeRewritePaths).map(to => `${to}**/*.js.map`),
```

Comment:
We'd want to keep this, it's useful if the pre-compilation step outputs source maps. And it should be harmless in case we run `tsc` ourselves.